eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/RegistryHandler.java

```diff
@@ -24,6 +24,7 @@
 import org.apache.eventmesh.runtime.admin.response.GetRegistryResponse;
 import org.apache.eventmesh.runtime.common.EventHttpHandler;
 import org.apache.eventmesh.runtime.registry.Registry;
+import org.apache.eventmesh.common.Constants;
```

Comment:
The location of this import should be here:
![image](https://user-images.githubusercontent.com/91315508/234743165-f915c336-6886-4d86-8dd2-4668825c14dc.png)